include/echoFunctions.php

```diff
@@ -114,6 +138,189 @@ function echoDialogBox(){
+
+	// if(){ //something w doc.getbyel? or need another parameter
+	//
+	// }
+	// $eventsGoingTo=getAllEventsThisUserIsGoingTo($userID);
+	$eventsGoingTo=get3EventsUserGoingTo($userID);
+
+	if ($eventsGoingTo == null) {
+		echo "Going to 0 events";
+	}
+	else {
+		foreach ($eventsGoingTo as $event) {
+		// var_dump($event);
+		echo "
+		<div class='row accountColumn' style='border:solid; border-color: gray;'>";
+
+		echoEvent($event);
+		echo "
+		</div><br>";
+		}
+	}
+}
+
+function echoAllEventsGoingTo($userID){
+
+	$eventsGoingTo=getAllEventsThisUserIsGoingTo($userID);
+
+	if ($eventsGoingTo == null) {
+		echo "Going to 0 events";
+	}
+	else {
+		foreach ($eventsGoingTo as $event) {
+		echo "
+		<div class='row accountColumn' style='border:solid; border-color: gray;'>";
+
+		echoEvent($event);
+		echo "
+		</div><br>";
+		}
+	}
+}
+
+
+function echoAllEventsCreated($userID){
+
+	$eventsCreated=getAllEventsCreated($userID);
+	if ($eventsCreated == null) {
+		echo "0 events created";
+	}
+	else{
+		foreach ($eventsCreated as $event) {
+			// var_dump($event);
+			echo "
+			<div class='row accountColumn' style='border:solid; border-color: gray;'>";
+			echoEvent($event);
+			echo "</div> <br>";
+		}
+	}
+}
+
+
+function echo3EventsCreated($userID){
+	$eventsCreated=get3EventsCreated($userID);
+	if ($eventsCreated == null) {
+		echo "0 events created";
+	}
+	else{
+		foreach ($eventsCreated as $event) {
+			// var_dump($event);
+			echo "
+			<div class='row accountColumn' style='border:solid; border-color: gray;'>";
+			echoEvent($event);
+			echo "</div> <br>";
+		}
+	}
+}
+
+
+
+
+function echoColumnEvent($eventID){
+	$thisEvent=getOneEvent($eventID);
+	echo "
+	<div class='trendingColumn'>";
+		echoEvent($thisEvent);
+	echo "
+	</div>";
+}
+
+function echoColumnTwoEvent($eventID){
+	$thisEvent=getOneEvent($eventID);
+	echo "<div class='trendingColumn' style='margin-left:5%; margin-right:5%;'>";
+		echoEvent($thisEvent);
+	echo "
+	</div>";
+}
+
+// $direction = "up";
+function echoUpVoteButton($eventID, $upVoted){
+	$direction = 'up'; //nic fixed this. very weird string stuff, but it works now!
+	if(!isset($_SESSION['userID'])){
+		$s = null;
+	}
+	else{
+		$s = $_SESSION['userID'];
+	}
+	echo"
+	<a href='javascript://' onclick=\"intakeVote($eventID, '$direction', $s);\">
+		<img id='upVoteButton_$eventID' class='iconAligned $upVoted' src='/pics/arrow2.jpg' alt='arrow' height=40px>
+	</a>";
+
+}
+
+function echoDownVoteButton($eventID, $downVoted){ //sohuld there be a ; after the function?
+	$direction = 'down';
+	if(!isset($_SESSION['userID'])){
+		$s = null;
+	}
+	else{
+		$s = $_SESSION['userID'];
+	}
+	echo"
+	<a href='javascript://' onclick=\"intakeVote($eventID, '$direction', $s);\">
+		<img id='downVoteButton_$eventID' class='iconAligned $downVoted' src='/pics/line2.jpg' alt='line' height=40px>
+	</a>";
+}
+
+
+
+function echoEvent($event){
+
+	//re calculate points, hourly
+	$hold = getLastCalculated2($event['eventID']);
```

Comment:
You already have the whole event here right? Can't you just pull the last calculated date from that?